Controllers/db/index.js

```diff
@@ -0,0 +1,12 @@
+const mysql = require('mysql');
+const pool = mysql.createPool({
+  connectionLimit: 10,
```

Comment:
This is the default, I guess you can omit this statement, we can increase the connectionLimit in future. Till then, we should work with default one.

Reply:
Ok